main/webapp/modules/core/styles/index/open-project-ui.less

```diff
@@ -111,113 +111,165 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 }
 
 #metadata-body table {
-  border-collapse:collapse
+  border-collapse: collapse;
+  table-layout: fixed;
 }
 
-#metadata-body td {
-    padding: 1px
+table.list-table{
+    margin:0;
 }
 
-#metadata-body tr:nth-child(even) {background: #f2f2f2}
-#metadata-body tr:nth-child(odd) {background: #FFF}
+#metadata-body td {
+  padding: 1px;
+}
 
-#metadata-body table {
-  table-layout: fixed;
+#metadata-body tr:nth-child(even) {
+  background: #f2f2f2;
+}
+#metadata-body tr:nth-child(odd) {
```

Comment:
@Babi-B: I wonder. I can't find this ID in the page: `#metadata-body`. Is it still existing?